README.md

```diff
@@ -198,14 +198,29 @@ pip install pyfstat[chainconsumer,pycuda,style]
 ```
 This command accepts the "development mode" tag `-e`.
 
-* If you prefer to make your own LALSuite installation
+### Using LALSuite built from source
+
+If you prefer to make your own LALSuite installation
 [from source](https://git.ligo.org/lscsoft/lalsuite/),
 make sure it is **swig-enabled** and contains at least the `lalpulsar` and `lalapps` packages.
-A minimal configuration line to use would be e.g.:
+
+Following the instructions from their README,
+at the step where it tells you to run `./configure`,
+an example minimal line would be e.g.:
+```
+./configure --prefix=${HOME}/lalsuite-install --disable-all-lal --enable-lalpulsar --enable-lalapps --enable-swig --enable-swig-python --disable-swig-octave
 ```
-./configure --prefix=${HOME}/lalsuite-install --disable-all-lal --enable-lalpulsar --enable-lalapps --enable-swig
+
+Then after the `make && make install` step,
+you can point your environment to this installation by doing
+```
+source ${HOME}/lalsuite-install/etc/lalsuite-user-env.sh
 ```
 
+You have to redo this in every new session,
+unless you tweak your venv / conda env activation script
+to do it automatically.
+
```

Comment:
Alternatively, Python's builtin [site](https://docs.python.org/3.9/library/site.html#module-site) can be used to direct the Python interpreter to the right LALSuite installation using `site-packages`. To do so, add the following minimal script to one of the available `site-packages` paths (which can be retrieved using `python -c 'import site; print( site.getsitepackages())'`):


```
usercustomize.py
----------------
import sys

custom_lalsuite = "${HOME}/lalsuite-install/lib/pythonX.Y/site-packages"
sys.path.insert(0, custom_lalsuite)
```

where `pythonX.Y` is dependent upon the Python version with which LALSuite was installed.